public/src/components/drawer.tsx

```diff
@@ -177,6 +177,11 @@ export default function NavDrawer(): React.ReactElement {
             <ListItemText primary="Channel Switches" />
           </ListItem>
         </Link>
+        <Link key="Campaigns" to="/campaigns" className={classes.link}>
+          <ListItem className={classes.listItem} button key="Campaigns">
+            <ListItemText primary="Campaigns" />
+          </ListItem>
+        </Link>
       </div>
```

Comment:
I was wondering if it would be better to put the 'Campaigns' link at the top of the list? Possibly under its own 'Campaigns management' heading?
![Screenshot 2022-03-07 at 12 15 30](https://user-images.githubusercontent.com/5357530/157033060-ac4fba5a-819b-4f5c-909f-3ec9780386b0.png)

Reply:
I'll move it to the top. Let's see where this goes before giving it its own section - it's currently a very simple feature